knowledge/tech_industry/microshift/attribution.txt

```diff
@@ -0,0 +1,4 @@
+Title of work:Microshift
+Link to work: https://www.redhat.com/en/topics/edge-computing/microshift
```

Comment:
I'm sorry but we are only taking Wikipedia submissions right now. Until we have clear direction from legal please put a PR with this domain in this: https://github.com/instructlab/dev-docs/blob/main/docs/knowledge-submissions-past-wikipedia.md